src/frontend/src/components/ActionsMenu.tsx

```diff
@@ -40,7 +40,19 @@ const ActionsMenu: React.FC<ActionsMenuProps> = ({ buttons, title = 'Actions' })
       >
         {title}
       </NERButton>
-      <Menu open={dropdownOpen} anchorEl={anchorEl} onClose={handleDropdownClose}>
+      <Menu
+        open={dropdownOpen}
+        anchorEl={anchorEl}
+        anchorOrigin={{
+          vertical: 'bottom',
+          horizontal: 'right'
+        }}
+        transformOrigin={{
+          vertical: 'top',
+          horizontal: 'right'
+        }}
+        onClose={handleDropdownClose}
+      >
```

Comment:
You don't need to make any of these changes

Reply:
Yeah, this was for another pr that was an accident lmao.

Reply:
How do I remove the push from the other issue? In this case #955 from January 25th